docs/integrations/sources/mssql.md

```diff
@@ -41,13 +35,14 @@ This is dependent on your networking setup. The easiest way to verify if Airbyte
 
 This step is optional but highly recommended to allow for better permission control and auditing. Alternatively, you can use Airbyte with an existing user in your database.
 
-_Coming soon: suggestions on how to create this user._
-
 #### 3. Your database user should now be ready for use with Airbyte!
 
+#### Airbyte Cloud
+On Airbyte Cloud, only secured connections to your MSSQL instance are supported in source configuration, Either by configuring you connection usign one of the supported SSL Methods or by using an SSH Tunnel.
```

Comment:
Spelling errors
On Airbyte Cloud, only secured connections to your MSSQL instance are supported in source configuration. You may either configure your connection using one of the supported SSL Methods or by using an SSH Tunnel.

Reply:
Thank you!